packages/block-editor/src/components/block-lock/toolbar.js

```diff
@@ -60,7 +60,7 @@ export default function BlockLockToolbar( { clientId, wrapperRef } ) {
 			<ToolbarGroup className="block-editor-block-lock-toolbar">
 				<ToolbarButton
 					icon={ lock }
-					label={ __( 'Unlock' ) }
+					label={ __( 'Unlock…' ) }
```

Comment:
Probably not super helpful for block toolbar buttons which are hidden by default.

I also noticed that the "Locking/Unlocking" items are missing the `aria-haspopup/aria-expanded` (#24796). I'll add those tomorrow.

It would be nice to get a11y feedback on this one.

Reply:
@alexstine thoughts on adding an ellipsis for the ToolbarButton text here?

Reply:
@richtabor It really doesn't change things much for screen readers. It will be highly depend it on how each screen reader decides to read "..." and this is also based on user controllable settings. Why not this for example?

```suggestion
					label={ __( 'Unlock, opens modal' ) }
```

The ARIA attributes, `aria-haspopup` should fix this for accessibility but not sure what you do here visually.